CONTRIBUTING.md

```diff
@@ -36,11 +36,11 @@ docker-compose up broker
 
 # Start node
 bin/server_build
-bin/server_run --node 123456
+bin/server_run --identity=0x123456..
```

Comment:
Why not omit identity parameter? It will create a new default identity and re-use the same identity later, which seems easier for contributors at this point :)

Reply:
Fixed